### PR TITLE
Super Hornet mod support 

### PR DESCRIPTION
**Pull request by Chilli935**
F/A-18E and F/A-18F are added to their factions, Growler is currently in its respective factions but was released in 2009, this will be changed if needed.

edit: correction on growler comment

### CODE REVIEW

File: resources/units/aircraft/FA-18E_super-hornet.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  The Super Hornet is also known for its extreme, slow-speed maneuverability in a dogfight.
+  Although incredibly deadly, the Super Hornet is also a very easy aircraft to fly.'
+introduced: 1999
+manufacturer: Boeing Defense, Space & Security
+origin: USA
+price: 28
+role: Carrier-based Multirole Fighter
+# DCS default livery for the Hornet is the Blue Angels.
+default_livery: "VFA-34"
+fuel:
```
Comment (DanAlbert) [inline resources/units/aircraft/FA-18E_super-hornet.yaml]: Does it really have identical fuel consumption as the legacy hornet? I find that surprising.

File: game/factions/faction.py [hunk 2]
```diff
@@ -300,6 +300,12 @@ def apply_mod_settings(self, mod_settings: ModSettings) -> None:
             self.remove_aircraft("Su-57")
         if not mod_settings.ov10a_bronco:
             self.remove_aircraft("Bronco-OV-10A")
+        if not mod_settings.fa18e_superhornet:
```
Comment (DanAlbert) [inline game/factions/faction.py]: Aren't these all the same mod? If so, just one checkbox.

File: resources/factions/australia_2005.yaml [hunk 3]
```diff
@@ -11,6 +11,9 @@ aircrafts:
   - F/A-18C Hornet (Lot 20)
   - SH-60B Seahawk
   - UH-1H Iroquois
+  - F/A-18E Super Hornet
+  - F/A-18F Super Hornet
+  - EA-18G Growler
```
Comment (DanAlbert) [inline resources/factions/australia_2005.yaml:16]: You're missing the banner, icon, and yaml file for the growler.

**Review by DanAlbert — COMMENTED** [review]
Aren't

File: resources/factions/australia_2005.yaml (hunk 3 above)
Comment (Chilli935) [reply resources/factions/australia_2005.yaml:16]: changed

Comment (Chilli935) [reply resources/factions/australia_2005.yaml:16]: banner and icon added

File: game/factions/faction.py (hunk 2 above)
Comment (Chilli935) [reply game/factions/faction.py]: checkboxes changed

File: resources/units/aircraft/FA-18E_super-hornet.yaml (hunk 1 above)
Comment (Chilli935) [reply resources/units/aircraft/FA-18E_super-hornet.yaml]: fuel consumption removed because I sadly do not have the free time to test the aircraft.

Comment (DanAlbert) [reply resources/units/aircraft/FA-18E_super-hornet.yaml]: Works for me. No data is better than bad data. Most aircraft already have no data anyway.

**Pull request merged**